tests/stdlib/trstgen.nim

```diff
@@ -201,14 +206,14 @@ not in table"""
         `|` outside a table cell should render as `\|`
     consistently with markdown, see https://stackoverflow.com/a/66557930/1426932
     ]#
-    doAssert output1 == """
+    check(output1 == """
```

Comment:
how about import std/strformat and using &"". {...}..""" to avoid breaking the string?